lab-matt/cowsay.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+const http = require('http');
+const parseUrl = require('url').parse;
+const parseQuery = require('querystring').parse;
+const cowsay = require('cowsay');
+
+const parseBody = require('./lib/parse-json');
+
+const port = process.argv[2] || 5000;
+
+const server = http.createServer(function(req, res){
```

Comment:
You have implemented the '/cowsay' resource, but lack a '/api/cowsay' resource as specified by the assignment.  They should function congruently.